packages/eslint-plugin-ecmascript-compat-example/eslint.config.js

```diff
@@ -0,0 +1,20 @@
+import compat from 'eslint-plugin-ecmascript-compat';
```

Comment:
[**The configuration for eslint-plugin-compat**](https://github.com/amilajack/eslint-plugin-compat#new-config-eslintconfigmjs):

```js
import compat from "eslint-plugin-compat";

```

I think it would be nice to use here a name other than `compat` to prevent conflicts with eslint-plugin-compat.

Thanks.